packages/bitcore-node/src/services/p2p/index.ts

```diff
@@ -84,68 +86,86 @@ export class P2pRunner {
     this.events = new EventEmitter();
   }
 
+  async wireupBlockStream(syncer, parent?) {
```

Comment:
these args need types:
```typescript
async wireupBlockStream(syncer: { add: (string) => void; }, parent?: { height: number; chain: string }): Observable<_>
```

Reply:
Also maybe make these methods `private`? It doesn't make sense to me for a user to be able to wire up something only partially, and to have to remember to do both, and maybe do it twice accidentally.